perllib/FixMyStreet/Cobrand/Peterborough.pm

```diff
@@ -514,9 +514,23 @@ sub has_pending_bulky_collection {
             state =>
                 { '=', [ FixMyStreet::DB::Result::Problem->open_states ] },
         },
-    )->all;
+    );
+}
+
+sub bulky_can_view_collection {
```

Comment:
Do we need this function at all now, or is the Report.pm code doing it for us already, it already has all the 'private = reporter or staff only' checks in.

Reply:
It's still being called from the `bin_days_bulky.html` template so still needed. Don't think it needs to be called from within `bulky_view` though, you're right.